scripts/install.sh

```diff
@@ -112,9 +112,8 @@ download_draft_cli_stable_version(){
     FILENAME="draft-$OS-$ARCH"
   fi
   log INFO "Starting Draft CLI Download for $FILENAME"
-  DRAFTCLIVERSION=$(curl -L -s https://api.github.com/repos/Azure/draft/releases/latest | jq -r '.tag_name')
-  log INFO "Starting Draft CLI Version $DRAFTCLIVERSION"
-  DRAFTCLIURL="https://github.com/Azure/draft/releases/download/$DRAFTCLIVERSION/$FILENAME"
+  log INFO "Draft CLI Version $DRAFT_CLI_VERSION"
+  DRAFTCLIURL="https://github.com/Azure/draft/releases/download/$DRAFT_CLI_VERSION/$FILENAME"
```

Comment:
how does this get set for the user? i see that we set it for the integration test, but wouldn't it be empty when a user runs the install script on their machine since they don't have the github action that sets the DRAFT_CLI_VERSION env?

Reply:
@davidgamero good catch, will take some more time to think about it.